File: 프로그래머스/0/120899. 가장 큰 수 찾기/가장 큰 수 찾기.js
function solution(array) {
    var answer = [];
    
    answer.push(array[0]);
    answer.push(array[1]);
    
    
    array.forEach((cur, idx) => 
            cur > answer[0] ? answer = [cur, idx] : answer = answer);
    
    
    return answer;
}

File: 프로그래머스/0/120899. 가장 큰 수 찾기/가장 큰 수 찾기.java
class Solution {
    public int[] solution(int[] array) {
        
        int idx = 0;
        int maxValue = array[0];
        
        for(var i = 0 ; i < array.length ; i++)
            if(maxValue < array[i]){
                idx = i;
                maxValue = array[i];
            }
        
        return new int[]{maxValue, idx};
    }
}